queries/hyprlang/highlights.scm

```diff
@@ -0,0 +1,60 @@
+(comment) @comment
+
+[
+  "source"
+  "exec"
+  "exec-once"
+] @keyword
+
+(keyword
+  (name) @keyword)
+
+(assignment
+  (name) @property)
+
+(section
+  (name) @module)
+
+(section
+  device: (device_name) @type)
+
+(variable) @variable
+
+(variable
+  "$" @punctuation.special)
```

Comment:
If `$` isn't used as a literal anywhere else, then you can just directly query it

```suggestion
"$" @punctuation.special
```

Reply:
I see, thanks for the tips!